.circleci/config.yml

```diff
@@ -61,21 +59,7 @@ jobs:
       - run:
           name: Verify requirements.txt file
           command: |
-            docker-compose run --rm --no-deps ci shell ./bin/run_verify_reqs.sh
-
-      - run:
-          name: Save image
-          # yamllint disable rule:line-length
-          command: |
-            mkdir -p workspace
-            docker save -o workspace/antenna-deploy-base.tar local/antenna_deploy_base:latest
-            gzip --fast workspace/antenna-deploy-base.tar
-          # yamllint enable rule:line-length
```

Comment:
This bit is left over from when we did parallel jobs in CI. We don't need to do this anymore.